modules/apps/digital-signature/digital-signature-test/src/testFunctional/macros/DigitalSignature.macro

```diff
@@ -1,5 +1,20 @@
 definition {
 
+	macro addRecipient {
+		for (var list : list ${addList}) {
+			AssertClick(
+				key_text = "Add Recipient",
+				locator1 = "Button#ANY_WITH_SPAN",
+				value1 = "Add Recipient");
+		}
+	}
+
+	macro addRecipientNotPresent {
+		AssertElementNotPresent(
+			key_text = ${notPresent},
+			locator1 = "Button#ANY_WITH_SPAN");
```

Comment:
Could you remove this variant: `${not present}` and put the same value you out above: `"Add Recipient"` so when you call this macro you won't need call the this variant.